src/mpilib20_init_finalise.F90

```diff
@@ -123,9 +129,9 @@ subroutine init_mpi_env(self, required_threading)
        call mpilib20_init(self)
     endif
 
-    call MPI_COMM_RANK(self%comm,  self%process ,    self%ierr)
-    call MPI_COMM_SIZE(self%comm,  self%n_processes, self%ierr)
-    call MPI_GROUP_SIZE(self%comm, self%group,       self%ierr)
+    call MPI_COMM_RANK(self%comm,   self%process ,    self%ierror)
+    call MPI_COMM_SIZE(self%comm,   self%n_processes, self%ierror)
+    call MPI_GROUP_SIZE(self%group, self%group_size,  self%ierror)
```

Comment:
I think this is superfluous. My understanding is that a group and comm should have the same size... could be wrong though. But certainly for COMM_WORLD, the group and comm are undoubtedly same size. 

- [ ] Leave as is for now

Reply:
Keep - makes more sense now I realise I should have included MPI_COMM_GROUP